packages/extension-kid/README.md

```diff
@@ -0,0 +1,141 @@
+# Extension Kid
```

Comment:
Please rename the package to `Extension Kit` – It's a funny name knowing the history of different code-names but without that knowledge it just looks like a misspelling.

Reply:
No.

Reply:
"No" is not an argument. If you want to stick with name you need to find a way to explain the history of the name.

Reply:
👶

Reply:
@nLight and I discussed the topic offline and agreed that with the added baby/child emoji in the title should mitigate the risk of someone thinking it's misspelled.